### PR TITLE
Notebook changes

### PR DESCRIPTION
**Pull request by jbednar**
This notebook is ready to merge, but I'll have more soon, so may as well treat this as a WIP PR.  The updates here to the NYC Taxi notebook are to simplify the explanations, tying each example to specific problems that it demonstrates and specific attempted solutions.

### CODE REVIEW

**Comment by jbednar** [comment]
The updated notebook is visible at https://anaconda.org/jbednar/nyc_taxi2/notebook .

File: examples/trajectory.ipynb [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "def create_image(x_range=x_range, y_range=y_range, w=500, h=500):\n",
+    "    cvs = ds.Canvas(x_range=x_range, y_range=y_range, plot_height=int(h), plot_width=int(w))\n",
```
Comment (jcrist) [inline examples/trajectory.ipynb]: If the javascript callback is fixed, these `int` calls should be removed.

File: examples/tseries.ipynb [hunk 2]
```diff
@@ -0,0 +1,323 @@
...
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "def create_image(x_range, y_range, w, h):\n",
+    "    cvs = ds.Canvas(x_range=x_range, y_range=y_range,\n",
+    "                    plot_height=int(h), plot_width=int(w))\n",
```
Comment (jcrist) [inline examples/tseries.ipynb]: same here.

File: examples/trajectory.ipynb (hunk 1 above)
Comment (jbednar) [inline examples/trajectory.ipynb]: At the moment those notebooks are on Anaconda Cloud, and if some one gets them from there they'll work with the current github master only if the int() calls are in there.  Once this PR is merged, then those calls will no longer be needed.  I guess I can remove them from the versions in this PR, as long as I don't later need to update Anaconda Cloud before the PR is merged...

Comment (jbednar) [inline examples/trajectory.ipynb]: Done.

**Pull request merged**